### ISSUE TITLE
QUESTION: How to update built-in component's traits?

### ISSUE THREAD

**Issue by Andrew-Chen-Wang** [artf/grapesjs#3322]
**Version:**

Latest

**Are you able to reproduce the bug from the demo?**

[ ] Yes
[X] No

**What is the expected behavior?**

Let's say you add the built-in link component to the canvas. If you head to the trait manager tab, then you can see the override traits when I do:
```
editor.DomComponents.addType('link', {
        model: {
            defaults: {
                traits: [
                    {name: "id"},
                    {
                        type: 'selectAdd',
                        name: 'href',
                        label: 'Link',
                    },
                    {name: "target"}
                ]
            }
        }
    });
editor.TraitManager.addType("selectAdd", {
createInput({ input }) {
// ...
}
})
```

on initialization. Reload the page and check the trait manager. It's come back to its default. But if you add the link block again, the updated trait manager is there. The old link block for some reason returns to default. How do I properly add traits then?

Basically follow this tutorial: https://grapesjs.com/docs/modules/Traits.html#define-new-trait-type

**Comment by Andrew-Chen-Wang** [comment]
So adding a trait is fine. But the trait itself is not reflected in components that are already loaded on page refresh or page load.

**Comment by Andrew-Chen-Wang** [comment]
nvm
